fix: make delete_locks more generic

stdavis opened the pull request.

stdavis committed:
fix: make delete_locks more generic

steveoh commented on src/swapper/swapper.py:
```diff
@@ -30,10 +33,10 @@ def delete_locks(fc_owner, fc_name, db_owner):
+        f'''SELECT * FROM SDE_process_information
+        WHERE SDE_ID IN(SELECT SDE_ID FROM SDE_table_locks
+        WHERE registration_id = (SELECT registration_id FROM SDE_table_registry
+        WHERE table_name = UPPER('{fc_name}') AND owner = UPPER('{fc_owner}')));
```
```suggestion
        WHERE UPPER(table_name) = UPPER('{fc_name}') AND UPPER(owner) = UPPER('{fc_owner}')));
```

stdavis replied:
I've been misunderstanding this part of a SQL query forever. I thought that this would have broken the query because the field names are lower case. I guess I need to think about it in terms of the field name will be substituted with the value.

steveoh reviewed (approved):
I'll assume the storage data is always uppercase but it seems safer to make sure we compare apples to apples

stdavis commented:
But actually, this is a bad PR. It breaks the queries for the production database. My local test database was using `dbo`, but it looks like it requires the `sde.` on SGID. I'll fix this tomorrow.

stdavis committed:
fix: add sde schema back to locks query
I originally tested this with a database that used dbo rather than sde and it worked. I'm guessing that it assumes dbo when you don't specify a schema.
